util/is-inverse-benchmark-record.js

```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * @param {{}} record
+ * @return {boolean}
+ */
+var isInverseBenchmarkRecord = function isInverseBenchmarkRecord(record) {
```

Comment:
Hm...I see that I did that elsewhere but I'm not sure what the best convention is. I'd say let's at least get rid of the repetition here so `var isInverseBenchmarkRecord = function(record) {...}`